[2021-12-08 10:16:37,167] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_meath.scrap_data scheduled__2021-12-07T04:15:00+00:00 [queued]>
[2021-12-08 10:16:37,271] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: scraping_daft_meath.scrap_data scheduled__2021-12-07T04:15:00+00:00 [queued]>
[2021-12-08 10:16:37,273] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 10:16:37,274] {taskinstance.py:1242} INFO - Starting attempt 1 of 3
[2021-12-08 10:16:37,275] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 10:16:37,377] {taskinstance.py:1262} INFO - Executing <Task(SparkSubmitOperator): scrap_data> on 2021-12-07 04:15:00+00:00
[2021-12-08 10:16:37,401] {standard_task_runner.py:52} INFO - Started process 151 to run task
[2021-12-08 10:16:37,730] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'scraping_daft_meath', 'scrap_data', 'scheduled__2021-12-07T04:15:00+00:00', '--job-id', '622', '--raw', '--subdir', 'DAGS_FOLDER/scraping_daft_meath.py', '--cfg-path', '/tmp/tmpasix6m8k', '--error-file', '/tmp/tmpzfbfe1c_']
[2021-12-08 10:16:37,733] {standard_task_runner.py:77} INFO - Job 622: Subtask scrap_data
[2021-12-08 10:16:38,701] {logging_mixin.py:109} INFO - Running <TaskInstance: scraping_daft_meath.scrap_data scheduled__2021-12-07T04:15:00+00:00 [running]> on host 041e6f8654c3
[2021-12-08 10:16:41,044] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=clemoni
AIRFLOW_CTX_DAG_ID=scraping_daft_meath
AIRFLOW_CTX_TASK_ID=scrap_data
AIRFLOW_CTX_EXECUTION_DATE=2021-12-07T04:15:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-07T04:15:00+00:00
[2021-12-08 10:16:41,134] {base.py:79} INFO - Using connection to: id: spark_default. Host: spark://spark:7077, Port: None, Schema: , Login: ***, Password: ***, extra: {}
[2021-12-08 10:16:41,179] {spark_submit.py:362} INFO - Spark-Submit cmd: spark-submit --master spark://spark:7077 --conf spark.master=spark://spark:7077 --name scrap data --verbose /opt/***/jobs/get_ads_per_county.py meath
[2021-12-08 10:17:46,327] {spark_submit.py:523} INFO - Using properties file: null
[2021-12-08 10:17:50,205] {spark_submit.py:523} INFO - WARNING: An illegal reflective access operation has occurred
[2021-12-08 10:17:50,208] {spark_submit.py:523} INFO - WARNING: Illegal reflective access by org.apache.spark.unsafe.Platform (file:/home/***/.local/lib/python3.6/site-packages/pyspark/jars/spark-unsafe_2.12-3.2.0.jar) to constructor java.nio.DirectByteBuffer(long,int)
[2021-12-08 10:17:50,210] {spark_submit.py:523} INFO - WARNING: Please consider reporting this to the maintainers of org.apache.spark.unsafe.Platform
[2021-12-08 10:17:50,222] {spark_submit.py:523} INFO - WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
[2021-12-08 10:17:50,233] {spark_submit.py:523} INFO - WARNING: All illegal access operations will be denied in a future release
[2021-12-08 10:17:56,459] {spark_submit.py:523} INFO - Parsed arguments:
[2021-12-08 10:17:56,474] {spark_submit.py:523} INFO - master                  spark://spark:7077
[2021-12-08 10:17:56,475] {spark_submit.py:523} INFO - deployMode              null
[2021-12-08 10:17:56,476] {spark_submit.py:523} INFO - executorMemory          null
[2021-12-08 10:17:56,477] {spark_submit.py:523} INFO - executorCores           null
[2021-12-08 10:17:56,507] {spark_submit.py:523} INFO - totalExecutorCores      null
[2021-12-08 10:17:56,509] {spark_submit.py:523} INFO - propertiesFile          null
[2021-12-08 10:17:56,513] {spark_submit.py:523} INFO - driverMemory            null
[2021-12-08 10:17:56,514] {spark_submit.py:523} INFO - driverCores             null
[2021-12-08 10:17:56,516] {spark_submit.py:523} INFO - driverExtraClassPath    null
[2021-12-08 10:17:56,516] {spark_submit.py:523} INFO - driverExtraLibraryPath  null
[2021-12-08 10:17:56,522] {spark_submit.py:523} INFO - driverExtraJavaOptions  null
[2021-12-08 10:17:56,524] {spark_submit.py:523} INFO - supervise               false
[2021-12-08 10:17:56,525] {spark_submit.py:523} INFO - queue                   null
[2021-12-08 10:17:56,526] {spark_submit.py:523} INFO - numExecutors            null
[2021-12-08 10:17:56,527] {spark_submit.py:523} INFO - files                   null
[2021-12-08 10:17:56,527] {spark_submit.py:523} INFO - pyFiles                 null
[2021-12-08 10:17:56,528] {spark_submit.py:523} INFO - archives                null
[2021-12-08 10:17:56,529] {spark_submit.py:523} INFO - mainClass               null
[2021-12-08 10:17:56,530] {spark_submit.py:523} INFO - primaryResource         file:/opt/***/jobs/get_ads_per_county.py
[2021-12-08 10:17:56,542] {spark_submit.py:523} INFO - name                    scrap data
[2021-12-08 10:17:56,544] {spark_submit.py:523} INFO - childArgs               [meath]
[2021-12-08 10:17:56,545] {spark_submit.py:523} INFO - jars                    null
[2021-12-08 10:17:56,546] {spark_submit.py:523} INFO - packages                null
[2021-12-08 10:17:56,552] {spark_submit.py:523} INFO - packagesExclusions      null
[2021-12-08 10:17:56,553] {spark_submit.py:523} INFO - repositories            null
[2021-12-08 10:17:56,554] {spark_submit.py:523} INFO - verbose                 true
[2021-12-08 10:17:56,555] {spark_submit.py:523} INFO - 
[2021-12-08 10:17:56,556] {spark_submit.py:523} INFO - Spark properties used, including those specified through
[2021-12-08 10:17:56,558] {spark_submit.py:523} INFO - --conf and those from the properties file null:
[2021-12-08 10:17:56,559] {spark_submit.py:523} INFO - (spark.master,spark://spark:7077)
[2021-12-08 10:17:56,573] {spark_submit.py:523} INFO - 
[2021-12-08 10:17:56,575] {spark_submit.py:523} INFO - 
[2021-12-08 10:18:22,772] {spark_submit.py:523} INFO - Main class:
[2021-12-08 10:18:22,893] {spark_submit.py:523} INFO - org.apache.spark.deploy.PythonRunner
[2021-12-08 10:18:22,906] {spark_submit.py:523} INFO - Arguments:
[2021-12-08 10:18:22,912] {spark_submit.py:523} INFO - file:/opt/***/jobs/get_ads_per_county.py
[2021-12-08 10:18:22,937] {spark_submit.py:523} INFO - null
[2021-12-08 10:18:22,941] {spark_submit.py:523} INFO - meath
[2021-12-08 10:18:22,946] {spark_submit.py:523} INFO - --verbose
[2021-12-08 10:18:22,949] {spark_submit.py:523} INFO - Spark config:
[2021-12-08 10:18:22,963] {spark_submit.py:523} INFO - (spark.app.name,scrap data)
[2021-12-08 10:18:22,978] {spark_submit.py:523} INFO - (spark.master,spark://spark:7077)
[2021-12-08 10:18:22,982] {spark_submit.py:523} INFO - (spark.submit.pyFiles,)
[2021-12-08 10:18:22,990] {spark_submit.py:523} INFO - (spark.submit.deployMode,client)
[2021-12-08 10:18:23,040] {spark_submit.py:523} INFO - Classpath elements:
[2021-12-08 10:18:23,042] {spark_submit.py:523} INFO - 
[2021-12-08 10:18:23,064] {spark_submit.py:523} INFO - 
[2021-12-08 10:18:23,065] {spark_submit.py:523} INFO - 
[2021-12-08 10:19:25,986] {spark_submit.py:523} INFO - Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
[2021-12-08 10:19:27,430] {spark_submit.py:523} INFO - 21/12/08 10:19:27 INFO SparkContext: Running Spark version 3.2.0
[2021-12-08 10:19:39,915] {spark_submit.py:523} INFO - 21/12/08 10:19:39 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2021-12-08 10:20:02,392] {spark_submit.py:523} INFO - 21/12/08 10:20:02 INFO ResourceUtils: ==============================================================
[2021-12-08 10:20:02,416] {spark_submit.py:523} INFO - 21/12/08 10:20:02 INFO ResourceUtils: No custom resources configured for spark.driver.
[2021-12-08 10:20:02,422] {spark_submit.py:523} INFO - 21/12/08 10:20:02 INFO ResourceUtils: ==============================================================
[2021-12-08 10:20:02,428] {spark_submit.py:523} INFO - 21/12/08 10:20:02 INFO SparkContext: Submitted application: Get Ads per county
[2021-12-08 10:20:02,873] {spark_submit.py:523} INFO - 21/12/08 10:20:02 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2021-12-08 10:20:03,706] {spark_submit.py:523} INFO - 21/12/08 10:20:03 INFO ResourceProfile: Limiting resource is cpu
[2021-12-08 10:20:03,760] {spark_submit.py:523} INFO - 21/12/08 10:20:03 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2021-12-08 10:20:09,802] {spark_submit.py:523} INFO - 21/12/08 10:20:09 INFO SecurityManager: Changing view acls to: ***
[2021-12-08 10:20:09,832] {spark_submit.py:523} INFO - 21/12/08 10:20:09 INFO SecurityManager: Changing modify acls to: ***
[2021-12-08 10:20:09,839] {spark_submit.py:523} INFO - 21/12/08 10:20:09 INFO SecurityManager: Changing view acls groups to:
[2021-12-08 10:20:09,840] {spark_submit.py:523} INFO - 21/12/08 10:20:09 INFO SecurityManager: Changing modify acls groups to:
[2021-12-08 10:20:09,870] {spark_submit.py:523} INFO - 21/12/08 10:20:09 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(***); groups with view permissions: Set(); users  with modify permissions: Set(***); groups with modify permissions: Set()
[2021-12-08 10:20:43,655] {spark_submit.py:523} INFO - 21/12/08 10:20:43 INFO Utils: Successfully started service 'sparkDriver' on port 34241.
[2021-12-08 10:20:58,072] {spark_submit.py:523} INFO - 21/12/08 10:20:56 INFO SparkEnv: Registering MapOutputTracker
[2021-12-08 10:21:19,133] {spark_submit.py:523} INFO - 21/12/08 10:21:17 INFO SparkEnv: Registering BlockManagerMaster
[2021-12-08 10:21:30,599] {spark_submit.py:523} INFO - 21/12/08 10:21:29 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2021-12-08 10:21:30,956] {spark_submit.py:523} INFO - 21/12/08 10:21:30 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2021-12-08 10:21:31,257] {spark_submit.py:523} INFO - 21/12/08 10:21:31 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2021-12-08 10:21:46,352] {spark_submit.py:523} INFO - 21/12/08 10:21:46 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-ef5c466a-b0fd-4c1b-9bbc-c72a45724683
[2021-12-08 10:21:57,526] {spark_submit.py:523} INFO - 21/12/08 10:21:57 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2021-12-08 10:21:57,987] {spark_submit.py:523} INFO - 21/12/08 10:21:57 INFO SparkEnv: Registering OutputCommitCoordinator
[2021-12-08 10:23:54,759] {spark_submit.py:523} INFO - 21/12/08 10:23:53 WARN Utils: Service 'SparkUI' could not bind on port 4040. Attempting port 4041.
[2021-12-08 10:23:54,793] {spark_submit.py:523} INFO - 21/12/08 10:23:54 WARN Utils: Service 'SparkUI' could not bind on port 4041. Attempting port 4042.
[2021-12-08 10:23:54,809] {spark_submit.py:523} INFO - 21/12/08 10:23:54 WARN Utils: Service 'SparkUI' could not bind on port 4042. Attempting port 4043.
[2021-12-08 10:23:54,815] {spark_submit.py:523} INFO - 21/12/08 10:23:54 WARN Utils: Service 'SparkUI' could not bind on port 4043. Attempting port 4044.
[2021-12-08 10:23:54,819] {spark_submit.py:523} INFO - 21/12/08 10:23:54 WARN Utils: Service 'SparkUI' could not bind on port 4044. Attempting port 4045.
[2021-12-08 10:23:54,829] {spark_submit.py:523} INFO - 21/12/08 10:23:54 WARN Utils: Service 'SparkUI' could not bind on port 4045. Attempting port 4046.
[2021-12-08 10:24:12,729] {spark_submit.py:523} INFO - 21/12/08 10:24:12 INFO Utils: Successfully started service 'SparkUI' on port 4046.
[2021-12-08 10:25:10,167] {base_job.py:230} ERROR - LocalTaskJob heartbeat got an exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2336, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/impl.py", line 241, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 508, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.6/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "postgres" to address: Temporary failure in name resolution


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.6/site-packages/airflow/jobs/base_job.py", line 202, in heartbeat
    session.merge(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 2171, in merge
    _resolve_conflict_map=_resolve_conflict_map,
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 2244, in _merge
    merged = self.query(mapper.class_).get(key[1])
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1018, in get
    return self._get_impl(ident, loading.load_on_pk_identity)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 1135, in _get_impl
    return db_load_fn(self, primary_key_identity)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/loading.py", line 286, in load_on_pk_identity
    return q.one()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3490, in one
    ret = self.one_or_none()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3459, in one_or_none
    ret = list(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3535, in __iter__
    return self._execute_and_instances(context)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3557, in _execute_and_instances
    querycontext, self._connection_from_session, close_with_result=True
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3572, in _get_bind_args
    mapper=self._bind_mapper(), clause=querycontext.statement, **kw
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/query.py", line 3550, in _connection_from_session
    conn = self.session.connection(**kw)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1145, in connection
    execution_options=execution_options,
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 1151, in _connection_for_bind
    engine, execution_options
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/orm/session.py", line 433, in _connection_for_bind
    conn = bind._contextual_connect()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2302, in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2340, in _wrap_pool_connect
    e, dialect, self
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1584, in _handle_dbapi_exception_noconnection
    sqlalchemy_exception, with_traceback=exc_info[2], from_=e
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2336, in _wrap_pool_connect
    return fn()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 364, in connect
    return _ConnectionFairy._checkout(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 778, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 495, in checkout
    rec = pool._do_get()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/impl.py", line 241, in _do_get
    return self._create_connection()
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 309, in _create_connection
    return _ConnectionRecord(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 440, in __init__
    self.__connect(first_connect_check=True)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 661, in __connect
    pool.logger.debug("Error on connect(): %s", e)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 70, in __exit__
    with_traceback=exc_tb,
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 182, in raise_
    raise exception
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 656, in __connect
    connection = pool._invoke_creator(self)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/strategies.py", line 114, in connect
    return dialect.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 508, in connect
    return self.dbapi.connect(*cargs, **cparams)
  File "/home/airflow/.local/lib/python3.6/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) could not translate host name "postgres" to address: Temporary failure in name resolution

(Background on this error at: http://sqlalche.me/e/13/e3q8)
[2021-12-08 10:25:59,757] {spark_submit.py:523} INFO - 21/12/08 10:25:59 INFO SparkUI: Bound SparkUI to 0.0.0.0, and started at http://041e6f8654c3:4046
[2021-12-08 10:27:44,060] {spark_submit.py:523} INFO - 21/12/08 10:27:42 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-08 10:27:54,437] {spark_submit.py:523} INFO - 21/12/08 10:27:54 INFO TransportClientFactory: Successfully created connection to spark/172.31.0.9:7077 after 2760 ms (0 ms spent in bootstraps)
[2021-12-08 10:28:07,533] {spark_submit.py:523} INFO - 21/12/08 10:28:07 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark:7077...
[2021-12-08 10:28:13,174] {spark_submit.py:523} INFO - 21/12/08 10:28:13 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20211208102813-0006
[2021-12-08 10:28:13,734] {spark_submit.py:523} INFO - 21/12/08 10:28:13 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 41047.
[2021-12-08 10:28:13,736] {spark_submit.py:523} INFO - 21/12/08 10:28:13 INFO NettyBlockTransferService: Server created on 041e6f8654c3:41047
[2021-12-08 10:28:13,744] {spark_submit.py:523} INFO - 21/12/08 10:28:13 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2021-12-08 10:28:14,013] {spark_submit.py:523} INFO - 21/12/08 10:28:14 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 041e6f8654c3, 41047, None)
[2021-12-08 10:28:14,544] {spark_submit.py:523} INFO - 21/12/08 10:28:14 INFO BlockManagerMasterEndpoint: Registering block manager 041e6f8654c3:41047 with 434.4 MiB RAM, BlockManagerId(driver, 041e6f8654c3, 41047, None)
[2021-12-08 10:28:14,637] {spark_submit.py:523} INFO - 21/12/08 10:28:14 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 041e6f8654c3, 41047, None)
[2021-12-08 10:28:14,662] {spark_submit.py:523} INFO - 21/12/08 10:28:14 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 041e6f8654c3, 41047, None)
[2021-12-08 10:28:24,346] {spark_submit.py:523} INFO - 21/12/08 10:28:24 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2021-12-08 10:28:24,500] {spark_submit.py:523} INFO - 21/12/08 10:28:24 INFO AsyncEventQueue: Process of event SparkListenerResourceProfileAdded(Profile: id = 0, executor resources: cores -> name: cores, amount: 1, script: , vendor: ,memory -> name: memory, amount: 1024, script: , vendor: ,offHeap -> name: offHeap, amount: 0, script: , vendor: , task resources: cpus -> name: cpus, amount: 1.0) by listener AppStatusListener took 1.036470575s.
[2021-12-08 10:28:26,674] {spark_submit.py:523} INFO - /home/***/.local/lib/python3.6/site-packages/pyspark/python/lib/pyspark.zip/pyspark/context.py:238: FutureWarning: Python 3.6 support is deprecated in Spark 3.2.
[2021-12-08 10:28:26,693] {spark_submit.py:523} INFO - FutureWarning
[2021-12-08 10:28:30,683] {spark_submit.py:523} INFO - 21/12/08 10:28:30 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2021-12-08 10:28:31,100] {spark_submit.py:523} INFO - 21/12/08 10:28:31 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2021-12-08 10:29:01,196] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:05,234] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:09,630] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:12,265] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:14,172] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:16,931] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:21,642] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:23,651] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:25,390] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:27,567] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:30,493] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:32,026] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:34,676] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:36,033] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:37,123] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:38,975] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:41,033] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:41,903] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:43,558] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:46,340] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:56,736] {spark_submit.py:523} INFO - running...
[2021-12-08 10:29:58,418] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:00,072] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:02,356] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:04,520] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:06,091] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:08,105] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:09,322] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:11,130] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:13,953] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:16,957] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:17,863] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:18,736] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:20,379] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:22,353] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:23,491] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:24,557] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:25,794] {spark_submit.py:523} INFO - running...
[2021-12-08 10:30:33,577] {spark_submit.py:523} INFO - 21/12/08 10:30:33 INFO SparkContext: Invoking stop() from shutdown hook
[2021-12-08 10:30:33,725] {spark_submit.py:523} INFO - 21/12/08 10:30:33 INFO SparkUI: Stopped Spark web UI at http://041e6f8654c3:4046
[2021-12-08 10:30:33,787] {spark_submit.py:523} INFO - 21/12/08 10:30:33 INFO StandaloneSchedulerBackend: Shutting down all executors
[2021-12-08 10:30:33,822] {spark_submit.py:523} INFO - 21/12/08 10:30:33 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2021-12-08 10:30:33,942] {spark_submit.py:523} INFO - 21/12/08 10:30:33 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2021-12-08 10:30:34,414] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO MemoryStore: MemoryStore cleared
[2021-12-08 10:30:34,422] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO BlockManager: BlockManager stopped
[2021-12-08 10:30:34,463] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO BlockManagerMaster: BlockManagerMaster stopped
[2021-12-08 10:30:34,469] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2021-12-08 10:30:34,601] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO SparkContext: Successfully stopped SparkContext
[2021-12-08 10:30:34,602] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO ShutdownHookManager: Shutdown hook called
[2021-12-08 10:30:34,604] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-7de4e761-cc79-409a-b326-4cb201441615
[2021-12-08 10:30:34,611] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-7de4e761-cc79-409a-b326-4cb201441615/pyspark-d50f62a3-2efa-4259-9000-8eb6a7f7d076
[2021-12-08 10:30:34,645] {spark_submit.py:523} INFO - 21/12/08 10:30:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-f7f92879-eb80-4084-a411-3c4fc8cdfafa
[2021-12-08 10:30:36,736] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=scraping_daft_meath, task_id=scrap_data, execution_date=20211207T041500, start_date=20211208T101637, end_date=20211208T103036
[2021-12-08 10:30:37,082] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-08 10:30:37,489] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
